iatistandard/root/index.html

```diff
@@ -88,10 +88,7 @@ <h3 class="assistive-text">Main menu</h3>
         <a href="202/">Version 2.02</a>
     </li>
     <li class="menu-item menu-item-type-post_type menu-item-object-page ">
-        <a href="201/">Version 2.01</a>
-    </li>
-    <li class="menu-item menu-item-type-post_type menu-item-object-page ">
-        <a href="upgrades/all-versions/">Other Versions</a>
+        <a href="202/upgrades/all-versions/">Other Versions</a>
```

Comment:
I think this was correct before, i.e. I think this should link to `upgrades/all-versions/` and not `202/upgrades/all-versions/`.